web/ui/static/js/targets.js

```diff
@@ -21,6 +21,15 @@ function showUnhealthy(_, container) {
 }
 
 function init() {
+  if (!localStorage.selectedTab || localStorage.selectedTab == "all-targets"){
+    $("div label:nth-child(1)").addClass("active");
```

Comment:
Maybe slightly better to find the labels by ID (`all-targets` / `unhealthy-targets`) and set the parent active?

Reply:
Done.